ospd/ospd_ssh.py

```diff
@@ -77,11 +77,13 @@ class OSPDaemonSimpleSSH(OSPDaemon):
     an array.
     """
 
-    def __init__(self, certfile, keyfile, cafile):
+    def __init__(self, certfile, keyfile, cafile, niceness):
```

Comment:
Just this single change is missing to make niceness optional here too
```suggestion
    def __init__(self, certfile, keyfile, cafile, niceness=None):
```